### PR TITLE
fix some documents

### PR DESCRIPTION
**Pull request by mottodora**

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -42,4 +42,18 @@ You can use ``setup.py`` to install Chainer from the tarball::
...
 
    $ git clone https://github.com/pfnet-research/chainerchem.git
-   $ pip install -e chainerchem
\ No newline at end of file
+   $ pip install -e chainerchem
+
+Run example training code
+========================
+`The official repository <https://github.com/pfnet-research/chainerchem>`_ provides examples
+several graph convolution networks with the Tox21 and QM9 datasets
```
Comment (corochann) [inline docs/source/install.rst]: provides examples
of training several graph convolution networks with the Tox21 and QM9 datasets

File: docs/source/install.rst [hunk 2]
```diff
@@ -42,4 +42,18 @@ You can use ``setup.py`` to install Chainer from the tarball::
...
+(the Tox21 example has inference code as well). You can obtain the code by cloning
+the repository::
```
Comment (corochann) [inline docs/source/install.rst]: The code can be obtained by cloning the repository... is better?

File: docs/source/install.rst [hunk 3]
```diff
@@ -42,4 +42,18 @@ You can use ``setup.py`` to install Chainer from the tarball::
...
    $ git clone https://github.com/pfnet-research/chainerchem.git
-   $ pip install -e chainerchem
\ No newline at end of file
+   $ pip install -e chainerchem
+
+Run example training code
+========================
+`The official repository <https://github.com/pfnet-research/chainerchem>`_ provides examples
+several graph convolution networks with the Tox21 and QM9 datasets
+(the Tox21 example has inference code as well). You can obtain the code by cloning
```
Comment (corochann) [inline docs/source/install.rst]: Just a comment that when we wrote specific current condition of the source code, we might need to modify docs frequently (When new dataset example is added we might need to update this sentence.)

Comment (mottodora) [reply docs/source/install.rst]: I will delete "with the Tox21 and QM9 datasets
+(the Tox21 example has inference code as well)"

**Pull request merged**